beaker-vue/src/components/chat-interface/ChatQueryCell.vue

```diff
@@ -0,0 +1,773 @@
+<template>
```

Comment:
As suggested, split out ChatQueryCell (chat-related changes) from BeakerQueryCell.

The ChatQueryCellEvent / BeakerQueryCellEvent updates are superior, since the chat version imports BeakerQueryCellEvent and passes in a slot override+some props to make BeakerQueryCellEvent agnostic to chat changes AND doesn't duplicate logic. I haven't explored doing the same for this file.

Reply:
@mattprintz What do you think? I wasn't successful on removing duplication yet. What suggestions/options do you have for this file?

Reply:
ie I could look into splitting and removing duplication. (I hope I interpreted the advice to split the file, to remove isChat, correctly).

Reply:
The above questions on great duplication between ChatQueryCell and BeakerQueryCell have been address on a commit after my comments.

I'll see if I can clean up some more redundant code, but at least on my side this is working and looking much better.
I may need to checkout `dev` and compare behaviors to ensure there are no regression problems.